bids/analysis/tests/test_transformations.py

```diff
@@ -73,24 +73,52 @@ def test_convolve(collection):
         rt.get_duration() * collection.sampling_rate
 
     # Test adapative oversampling computation
+    args = (mock.ANY, 'spm', mock.ANY)
+    kwargs = dict(fir_delays=None, min_onset=0)
     with mock.patch('bids.analysis.transformations.compute.hrf') as mocked:
+        # Sampling rate is 10Hz, no oversampling needed
         transform.Convolve(collection, 'RT', output='rt_mock')
-        mocked.compute_regressor.assert_called_with(
-            mock.ANY, 'spm', mock.ANY, fir_delays=None, min_onset=0,
-            oversampling=1.0)
+        mocked.compute_regressor.assert_called_with(*args, oversampling=1.0, **kwargs)
 
     with mock.patch('bids.analysis.transformations.compute.hrf') as mocked:
+        # Sampling rate is 10Hz, no oversampling needed
         transform.Convolve(collection, 'rt_dense', output='rt_mock')
-        mocked.compute_regressor.assert_called_with(
-            mock.ANY, 'spm', mock.ANY, fir_delays=None, min_onset=0,
-            oversampling=3.0)
+        mocked.compute_regressor.assert_called_with(*args, oversampling=1.0, **kwargs)
```

Comment:
I don't really understand why this one was originally 3.

Reply:
I honestly have racked my brain and can't remember why it was set that way. My guess is that it was just copied over, and/or erring on the safe side. 

If your logic is able to handle high frequency events and appropriately oversample as not to lose any information, then I think we can safely ignore this.